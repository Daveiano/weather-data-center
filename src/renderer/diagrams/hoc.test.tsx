import React from 'react'
import { render } from '@testing-library/react'

import data from "../../../tests/data/scaling-input.json";
import config from "../../../tests/data/config.json";

import TemperatureBase from "./temperature/temperature-base";

test('hoc no render when data is not available', async () => {
  const { container } = render(
    <TemperatureBase
      title="Temperature"
      height="300px"
      data={data.map(item => ({
        id: item.id,
        _id: item._id,
        timeParsed: item.timeParsed,
        time: item.time,
        rain: item.rain
      }))}
      property="temperature"
      config={config}
    />
  );

  expect(container).toMatchSnapshot();
});