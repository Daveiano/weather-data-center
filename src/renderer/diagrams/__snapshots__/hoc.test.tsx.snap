// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hoc no render when data is not available 1`] = `<div />`;
